pages/governance/pipeline_templates.md

```diff
@@ -0,0 +1,60 @@
+# Pipeline templates
+
+> 📘 Enterprise feature
+> Pipeline templates is only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+## Overview
+
+Pipeline templates allow you to define standard pipeline step configurations to be used by all the pipelines in your organization.
+
+When a pipeline has a pipeline template assigned to it, the step configuration for that pipeline is inherited from the template and it is no longer possible to configure a custom step configuration for that pipeline via the UI.
+
+## Creating a pipeline template
+
+Only administrators can create or update pipeline templates.
+
+1. Open your Buildkite _Organization Settings_ and choose [Templates](https://buildkite.com/organizations/-/pipeline-templates)
+1. If this is your first template, click _Create a Template_. Otherwise choose _New Template_. A pipeline template is created automatically for you.
+1. Update the name and description of your new template and click _Apply_ to save your changes.
+1. You can also click _Edit_ to change the step configuration for the template.
+
+An administrator can add multiple templates to be used across the organization. Making changes and saving a template will apply those changes to all pipelines using that template.
+
+## Testing a pipeline template
+
+At any time an administrator may test a pipeline template against a pipeline via the _New Build_ button on the pipeline page.
+
+If a template exists for the organization then it can be selected from the _Pipeline template_ dropdown to create a new build using the step configuration from that template.
```

Comment:
Missing comma:

```suggestion
If a template exists for the organization, it can be selected from the _Pipeline template_ dropdown to create a new build using the step configuration from that template.
```